pkgs/by-name/my/myrescue/package.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchurl
+, installShellFiles
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "myrescue";
+  version = "0.9.8";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/project/myrescue/myrescue/myrescue-${finalAttrs.version}/myrescue-${finalAttrs.version}.tar.gz";
+    hash = "sha256-tO9gkDpEtmySatzV2Ktw3eq5SybCUGAUmKXiSxnkwdc=";
+  };
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  sourceRoot = "./src";
+
+  patches = [
+    ./0001-darwin-build-fixes.patch
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin
+    cp myrescue $out/bin/
```

I've seen several packages use install(1) and it seems this package would benefit from its use too:

```suggestion
    install -Dm755 myrescue -t $out/bin
```

---

@a-n-n-a-l-e-e in general, should we use `mkdir/cp` or `install`?

---

i use install when it makes sense. not sure if there is a policy one way or the other.